File: Assignment_41/Program2.c
/*
Write a recursive program which display below pattern
Input : 5
Output : 1 2 3 4 5
*/

#include <stdio.h>

void Display(int iNo)
{
    static int i = 1;

    if(iNo < 0)
    {
        iNo = -iNo;
    }
    
    if(i <= iNo)
    {
        printf(" %d\t",i);
        i++;
        Display(iNo);
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number : ");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

File: Assignment_46/Program2.java
/*
Write java program which accept number of rows and number of columns from user and display below pattern.
Input : iRow = 4, iCol = 4
Output : A      B       C       D
         a      b       c       d
         A      B       C       D
         a      b       c       d
*/          

import java.util.*;

class Pattern
{
    public void Pattern(int iRow, int iCol)
    {
        int i = 0, j = 0;
        char ch1 = '\0', ch2 = '\0';

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1, ch1 = 'A', ch2 = 'a'; j <= iCol; j++, ch1++, ch2++)
            {
                if((i % 2) == 0)
                {
                    System.out.print(ch2+"\t");
                }
                else
                {
                    System.out.print(ch1+"\t");
                }
            }
            System.out.println();
        }   
    }
}               

class Program2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0, iValue2 = 0;

        System.out.println("Enter number of rows : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter number of columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Pattern(iValue1, iValue2);
    }
}